<launch>
    <node name="mpu9250" pkg="mpu9250" type="mpu9250" output="log">
        <!--param name="config_file" type="string" value="$(arg config_path)" /-->
        <!--param name="vins_folder" type="string" value="$(arg vins_path)" /-->
    </node>
</launch>
